lib/runners/go.js

```diff
@@ -53,6 +53,8 @@ func Test${name}(t *testing.T) {
         // exit status 1
         // FAIL	{dirname of test file}	{seconds}s
         sanitizeStdOut: function(stdout) {
+            // special case when build failed
+            if (stdout.includes('[build failed]')) return '<ERROR::>Build Failed\n';
```

Comment:
Does any other information get written out when this happens? If so then maybe `stdout.replace('[build failed]', "\n<ERROR::>Build Failed\n")` would be better?